src/Excel.php

```diff
@@ -176,7 +176,7 @@ protected function export($export, string $fileName, string $writerType = null)
      *
      * @return string|null
      */
-    protected function findTypeByExtension($fileName, string $type = null): string
```

Comment:
Method is used in `protected function export($export, string $fileName, string $writerType = null)` too. There it expects a string. Can you throw the `NoTypeDetectedException` at the places where this method is used for exports.